Simplified verbose setup instructions (#4631)
GoogleChrome/developer.chrome.com: site/en/docs/privacy-sandbox/attribution-reporting-debugging/part-2/index.md

While there are numerous types of verbose debug reports, it's sufficient to check your verbose debugging setup with only **one type** of verbose debug report. If this one type of verbose debug report is correctly generated and received, this means that all types of verbose debug reports will be correctly generated and received as well, because all verbose debug reports use the same configuration and are sent to the same endpoint.

{% Aside 'gotchas' %}
Pick a verbose debug report that's simple to test for. `trigger-no-matching-source` is a good candidate, because you can generate it by simply converting. Verbose reports that relate to trigger events tend to be better candidates; verbose report types that relates to source events are more tedious to purposefully generate.
{% endAside %}

1. Open `chrome://attribution-internals` in your browser.

> 2. Using the site you have Attribution Reporting set up on, trigger an attribution (convert). Given that there was no ad engagement (impression or click) before that conversion, you should expect a verbose debug report of type `trigger-no-matching-source` to be generated.

Reviewer: ```suggestion
2. Trigger an attribution (convert) on your site that's set up with Attribution Reporting. Given that there was no ad engagement (impression or click) before this conversion, you should expect a verbose debug report of type `trigger-no-matching-source` will be generated.
```
